tools/topology/topology2/include/pipelines/cavs/dai-copier-be.conf

```diff
@@ -72,6 +72,7 @@ Class.Pipeline."dai-copier-be" {
 		copier."2" {
 			type	buffer
 			copier_type	"module"
+			node_type	$BUFFER_COPIER
```

Comment:
why do we need a dai-copier-be in the first place? or is this again a case of a pipeline name not describing well what it actually does?